python/gym_ignition/rbd/utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+import numpy as np
+
+
+def hat(vector3: np.ndarray) -> np.ndarray:
+
+    if vector3.size != 3:
+        raise ValueError(vector3)
+
+    s = np.zeros(shape=(3, 3))
+    s[1, 0] = vector3[2]
+    s[0, 1] = -vector3[2]
+    s[0, 2] = vector3[1]
+    s[2, 0] = -vector3[1]
+    s[2, 1] = vector3[0]
+    s[1, 2] = -vector3[0]
+
+    return s
+
+
+def vee(matrix3x3: np.ndarray) -> np.ndarray:
+
+    if matrix3x3.shape != (3, 3):
+        raise ValueError(skew)
+
+    skew_symmetric = skew(matrix3x3)
+
+    return np.array([skew_symmetric[2, 1],
+                     skew_symmetric[0, 2],
+                     skew_symmetric[1, 0]])
+
+
+def skew(matrix: np.ndarray) -> np.ndarray:
```

Comment:
I am afraid that this may be confusing to user, as they could expect that this function takes a 3\times 1 vector and return the 3x3 skew symmetric matrices, what now is done by the hat operator.

Reply:
Do you have any better name for the decomposition of a matrix in its skew symmetric and symmetric parts?

Reply:
`extract_skew` and `extract_symmetric` or `skew_part` and `symmetric_part` seems to be more clear to me, especially if no docs are provided.